scripts/upgrade

```diff
@@ -31,6 +31,9 @@ admin_pwd=$(ynh_app_setting_get --app=$app --key=admin_pwd)
 #=================================================
 
 upgrade_type=$(ynh_check_app_version_changed)
+# Workaround for now to force all instances to get the latest LDAP plugin
+# in order to create the dedicated tables
+upgrade_type="UPGRADE_APP"
```

Comment:
I'd surround this line by an `if the current version is the problematic version` but it's not really important

Reply:
Indeed, but I will remove it when rebasing the 2.10.2 update PR that came at the right time!